Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -397,11 +399,11 @@ public final class PackageInfoMapper: PackageInfoMapping {
     }
 
     fileprivate class func sanitize(targetName: String) -> String {
-        targetName.replacingOccurrences(of: ".", with: "_")
+        targetName.replacingOccurrences(of: ".", with: "_").replacingOccurrences(of: "-", with: "_")
```

Comment:
have you found this problem in some dependency?

Reply:
This was added because I am using the TargetName as the product name now, which for example is causing for `gRPC-cpp` the `TargetLinter` to complain